webview/src/shared/components/tooltip/Tooltip.tsx

```diff
@@ -6,6 +6,27 @@ import 'tippy.js/dist/tippy.css'
 export const HEADER_TOOLTIP_DELAY = 100
 export const CELL_TOOLTIP_DELAY = 1000
 
+const hideOnEsc = {
```

Comment:
Found this [here](https://github.com/atomiks/tippyjs/issues/750). It is also shown in the official docs: https://atomiks.github.io/tippyjs/v6/plugins/#hideonesc

Reply:
I can't think of a reason not to have this on any of the tooltips/context menus but LMK if you think differently.